concent_api/utils/tests/test_helpers.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+
+from django.test            import TestCase
+from django.utils           import timezone
+
+from utils.helpers          import parse_datetime_to_timestamp
+
+
+class HelpersTestCase(TestCase):
+
+    def test_parse_datetime_to_timestamp_should_return_correct_utc_timestamp(self):
+        """ Tests if parse_datetime_to_timestamp function works as expected. """
+
+        timestamp = 946684800  # 2000-01-01 00:00
+        assert datetime.datetime.fromtimestamp(timestamp) == datetime.datetime(2000, 1, 1, 0, 0)
+
+        for date_time in [
+            datetime.datetime(2000, 1, 1, 0, 0),
+            datetime.datetime(2000, 1, 1, 4, 37,   tzinfo = timezone.pytz.timezone('Asia/Kabul')),
+            datetime.datetime(1999, 12, 31, 19, 4, tzinfo = timezone.pytz.timezone('US/Eastern')),
```

Comment:
Please test with a `datetime` that has `tzinfo = timezone.utc` too. That's the most common case for us after all :)

Reply:
:+1: